.github/workflows/e2e-upgrade.yml

```diff
@@ -184,12 +216,77 @@ jobs:
           awsOpenSearchPwd: ${{ secrets.AWS_OPENSEARCH_PWD }}
           clusterCreation: "cli"
 
-      - name: Build CLI
-        uses: ./.github/actions/build_cli
+      - name: Remove Terraform plugin cache
+        if: always()
+        run: |
+          rm -rf constellation-terraform/.terraform
+          rm -rf constellation-iam-terraform/.terraform
+
+      - name: Upload Working Directory
+        if: always()
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
```

Comment:
Why not zip the working directory and upload as a single file?

Reply:
I wanted to have a small archive, zipping the directory would just add extra steps here, as we would have to delete all the files we don't want to zip & upload (e.g. binaries) and then unpack the archive afterwards